src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -352,7 +331,7 @@ private static Hashtable GetEditorTable(Type editorBaseType)
                 // actually run.
                 //
                 System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor(editorBaseType.TypeHandle);
-                table = s_editorTables[editorBaseType];
+                table = EditorTables[editorBaseType];
```

Comment:
None of these need to change from s_editorTables to EditorTables: we know it's already been initialized because it was accessed above.  LazyInitialized.EnsureInitialized should be very cheap, but it can't be any cheaper than just going to the field directly. (We could even cache it in a local, but for the purposes of this change, I think it's best to minimize what's touched to the minimal necessary.)